<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowToolBar</class>
 <widget class="QMainWindow" name="MainWindowToolBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindowToolBar</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCheckBox" name="check_floatable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>floatable</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_movable">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>moveable</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_area">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Qt::LeftToolBarArea</string>
     </property>
     <property name="checkState">
      <enum>Checked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Qt::RightToolBarArea</string>
     </property>
     <property name="checkState">
      <enum>Checked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Qt::TopToolBarArea</string>
     </property>
     <property name="checkState">
      <enum>Checked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Qt::BottomToolBarArea</string>
     </property>
     <property name="checkState">
      <enum>Checked</enum>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="listWidget_button">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Qt::ToolButtonIconOnly</string>
     </property>
     <property name="checkState">
      <enum>Checked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Qt::ToolButtonTextOnly</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Qt::ToolButtonTextBesideIcon</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Qt::ToolButtonTextUnderIcon</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>495</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionAction1"/>
    <addaction name="actionAction2"/>
   </widget>
   <addaction name="menufile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAction1"/>
   <addaction name="actionAction2"/>
  </widget>
  <action name="actionAction1">
   <property name="text">
    <string>Action1</string>
   </property>
  </action>
  <action name="actionAction2">
   <property name="text">
    <string>Action2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
